web_src/js/index.js

```diff
@@ -585,6 +585,35 @@ function initInstall() {
   });
 }
 
+function initIssueComments() {
+  if ($('.repository.view.issue .comments').length === 0) return;
+
+  $(document).click((event) => {
+    const urlTarget = $(':target');
+    if (urlTarget.length === 0) return;
+
+    const urlTargetId = urlTarget.attr('id');
+    if (!urlTargetId) return;
+    if (!/^(issue|pull)(comment)?-\d+$/.test(urlTargetId)) return;
+
+    const $target = $(event.target);
+
+    if ($target.closest(`#${urlTargetId}`).length === 0) {
+      if (!window.location.hash) return;
+
+      if (window.location.hash.length > 0 && window.location.hash !== '#') {
+        const i = window.location.toString().indexOf('#');
+
+        if (i >= 0) {
```

Comment:
Instead of nesting `if` so deeply, I'd suggest the pattern `if (cond) return` here to keep the code flat.

Reply:
I removed some checks. I think they are unnecessary because if there is an element with :target selected they should be. Am i wrong?